doc/contributor/howto-guide-adding-generated-libraries.md

```diff
@@ -280,11 +282,50 @@ index c4ce00489..1858b48dc 100755
 
 </details>
 
-## Commit these changes
+### Commit these changes
 
 ```shell
 git commit -m"Manually update READMEs, quickstart, and top-level stuff" .
 ```
 
+## Expanding a library
```

Comment:
Should we have a `howto-guide-expanding-a-generated-library.md` document instead?

Reply:
I am indifferent.